### ISSUE TITLE
Missing dir and content - "Models"

### ISSUE THREAD

**Issue by robtow** [vyassu/DeepSentiment#3]
Hi,

I added <Test-TrainingData_SVM.csv>, and things went a bit further - but there seems to be some more missing content from the repo - a dir named <Models>, and some expected content inside it:

`rob@ubuntu:~/Desktop/DeepSentiment/DeepSentiment$ ./script.sh 
mkdir: cannot create directory ‘Data’: File exists
mkdir: cannot create directory ‘inputdata’: File exists
/usr/local/lib/python2.7/dist-packages/sklearn/cross_validation.py:44: DeprecationWarning: This module was deprecated in version 0.18 in favor of the model_selection module into which all the refactored classes and functions are moved. Also note that the interface of the new CV iterators are different from that of this module. This module will be removed in 0.20.
  "This module will be removed in 0.20.", DeprecationWarning)
 * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
 * Restarting with stat
/usr/local/lib/python2.7/dist-packages/sklearn/cross_validation.py:44: DeprecationWarning: This module was deprecated in version 0.18 in favor of the model_selection module into which all the refactored classes and functions are moved. Also note that the interface of the new CV iterators are different from that of this module. This module will be removed in 0.20.
  "This module will be removed in 0.20.", DeprecationWarning)
 * Debugger is active!
 * Debugger pin code: 293-235-160
127.0.0.1 - - [27/Jan/2017 15:48:55] "OPTIONS /deepsentifile HTTP/1.1" 200 -
('Pitch and Loudness processing Start time:', datetime.time(15, 49, 0, 193969))
('SVM Model creation start:', datetime.time(15, 49, 0, 194061))
('SVM MOdel creation end:', datetime.time(15, 49, 0, 199762))
('File Data load End time:', datetime.time(15, 49, 0, 214853))
('Data preprocessing Start time:', datetime.time(15, 49, 0, 214917))
('Data preprocessing End time:', datetime.time(15, 49, 0, 214931))
('SVM Training started:', datetime.time(15, 49, 0, 258340))
/usr/local/lib/python2.7/dist-packages/sklearn/externals/joblib/parallel.py:540: UserWarning: Multiprocessing backed parallel loops cannot be nested below threads, setting n_jobs=1
  **self._backend_args)
('score', 1.0)
127.0.0.1 - - [27/Jan/2017 15:49:00] "POST /deepsentifile HTTP/1.1" 500 -
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1994, in __call__
    return self.wsgi_app(environ, start_response)
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1985, in wsgi_app
    response = self.handle_exception(e)
  File "/usr/local/lib/python2.7/dist-packages/flask_cors/extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1540, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python2.7/dist-packages/flask_cors/extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/rob/Desktop/DeepSentiment/DeepSentiment/WebInterface/webservice.py", line 73, in fileservice
    resultData = Controller.main(absfile)
  File "/home/rob/Desktop/DeepSentiment/DeepSentiment/Code/StandAlone/Controller.py", line 22, in main
    emotions = sp.main(filename)
  File "/home/rob/Desktop/DeepSentiment/DeepSentiment/Code/StandAlone/SpeechNetSVMMulticlass.py", line 159, in main
    svmnnet.trainNNet(ftrain,ltrain,feature_name=feature)
  File "/home/rob/Desktop/DeepSentiment/DeepSentiment/Code/StandAlone/SpeechNetSVMMulticlass.py", line 74, in trainNNet
    joblib.dump(self.model,  self.working_directory+filename)
  File "/usr/local/lib/python2.7/dist-packages/sklearn/externals/joblib/numpy_pickle.py", line 482, in dump
    with open(filename, 'wb') as f:
**IOError: [Errno 2] No such file or directory: '/home/rob/Desktop/DeepSentiment/DeepSentiment/Models/SVM_a.pkl'**

`

**Comment by vyassu** [comment]
Hey Rob,

I have uploaded the missing files and ironed out some of the issues within the Python and script files. Please download the latest code from the repository.

Regards
Vyas

**Comment by robtow** [comment]
What is the PYTHONPATH for your project?

**Comment by robtow** [comment]
Hi,  great work. The command line version works nicely. 

I tested using a Star Trek WAV file :Doohan: "Captain, I checked the engines. The warp drive, it's a hopeless pile o' junk!" (The Doomsday Machine)". I found it here (and converted it to the correct format & sample rate): <http://www.rosswalker.co.uk/tv_sounds/star_trek_p2.htm>

The web interface still has a glitch, on the posting of the results:

rob@ubuntu:~/Desktop/DeepSentiment/DeepSentiment/Code/StandAlone$ ./script.shWebservice started!! Use URL http://localhost:5000/deepsentiment
/usr/local/lib/python2.7/dist-packages/sklearn/cross_validation.py:44: DeprecationWarning: This module was deprecated in version 0.18 in favor of the model_selection module into which all the refactored classes and functions are moved. Also note that the interface of the new CV iterators are different from that of this module. This module will be removed in 0.20.
  "This module will be removed in 0.20.", DeprecationWarning)
INFO:werkzeug: * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO:werkzeug: * Restarting with stat
/usr/local/lib/python2.7/dist-packages/sklearn/cross_validation.py:44: DeprecationWarning: This module was deprecated in version 0.18 in favor of the model_selection module into which all the refactored classes and functions are moved. Also note that the interface of the new CV iterators are different from that of this module. This module will be removed in 0.20.
  "This module will be removed in 0.20.", DeprecationWarning)
WARNING:werkzeug: * Debugger is active!
INFO:werkzeug: * Debugger pin code: 293-235-160
INFO:werkzeug:127.0.0.1 - - [30/Jan/2017 05:30:10] "GET /deepsentiment HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [30/Jan/2017 05:30:10] "GET /static/css/deepUI.css HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [30/Jan/2017 05:30:10] "GET /static/js/deepui.js HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [30/Jan/2017 05:30:10] "GET /static/img/featr.jpg HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [30/Jan/2017 05:30:10] "GET /static/img/recoder_icon.jpg HTTP/1.1" 200 -
INFO:root:Loaded the Speech Recognition Module
INFO:root:Saved the input signal into WAVE File
INFO:root:Converted Speech To Text
INFO:root:Training of Model Disabled
INFO:root:Finished Loading SpeechNetSVMMulticlass Module
INFO:root:Finished Loading Test-Train data from file
/usr/local/lib/python2.7/dist-packages/sklearn/base.py:315: UserWarning: Trying to unpickle estimator LabelBinarizer from version pre-0.18 when using version 0.18.1. This might lead to breaking code or invalid results. Use at your own risk.
  UserWarning)
/usr/local/lib/python2.7/dist-packages/sklearn/base.py:315: UserWarning: Trying to unpickle estimator SVC from version pre-0.18 when using version 0.18.1. This might lead to breaking code or invalid results. Use at your own risk.
  UserWarning)
/usr/local/lib/python2.7/dist-packages/sklearn/base.py:315: UserWarning: Trying to unpickle estimator OneVsRestClassifier from version pre-0.18 when using version 0.18.1. This might lead to breaking code or invalid results. Use at your own risk.
  UserWarning)
INFO:root:Emotion Data: {'Angry': 1.0}
INFO:root:Retrieving the Training-Test Data for Speech2Text Model
INFO:root:Retrieving Validation Data for Speech2Text Model
INFO:root:Configuring the SVM Model
/usr/local/lib/python2.7/dist-packages/sklearn/externals/joblib/parallel.py:540: UserWarning: Multiprocessing backed parallel loops cannot be nested below threads, setting n_jobs=1
  **self._backend_args)
INFO:root: Finished Loading the saved Speech2Text Model.
INFO:root:Final Emotion set:: {'Angry': 1.0, 'data': 'that machete asians that was at how hopeless at junk'}
INFO:werkzeug:127.0.0.1 - - [30/Jan/2017 05:30:34] "POST /deepsentifile HTTP/1.1" 200 -
ERROR:werkzeug:Error on request:
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/werkzeug/serving.py", line 205, in run_wsgi
    execute(self.server.app)
  File "/usr/local/lib/python2.7/dist-packages/werkzeug/serving.py", line 196, in execute
    write(data)
  File "/usr/local/lib/python2.7/dist-packages/werkzeug/serving.py", line 164, in write
    self.send_header(key, value)
  File "/usr/lib/python2.7/BaseHTTPServer.py", line 401, in send_header
    self.wfile.write("%s: %s\r\n" % (keyword, value))
IOError: [Errno 32] Broken pipe
